src/classes/Table.ts

```diff
@@ -8,32 +6,162 @@
 
 // TODO: Check duplicate entity names code
 
-// Import parseTable 
-const parseTable = require('../lib/parseTable')
-const parseFilters = require('../lib/expressionBuilder')
-const parseProjections = require('../lib/projectionBuilder')
-const validateTypes = require('../lib/validateTypes')
+// Import libraries, types, and classes
+import Entity from './Entity'
+import DynamoDb, { DocumentClient } from 'aws-sdk/clients/dynamodb'
+import { parseTable, ParsedTable } from '../lib/parseTable'
+import parseFilters from '../lib/expressionBuilder'
+import validateTypes from '../lib/validateTypes'
+import { FilterExpressions } from '../lib/expressionBuilder'
+import parseProjections, { ProjectionAttributes, ProjectionAttributesTable } from '../lib/projectionBuilder'
+import { ParsedEntity } from '../lib/parseEntity'
 
 // Import standard error handler
-const { error } = require('../lib/utils')
-
-// Import Entity class
-const Entity = require('./Entity')
-
-// Define condition error
-const conditonError = (op) =>
-  error(`You can only supply one sortKey condition per query. Already using '${op}'`)
+import { error, conditonError } from '../lib/utils'
+import { Document } from 'aws-sdk/clients/textract'
```

Comment:
AWS SDK is not a dependency in your package.json (it's a dev dependency). 

Do you want it to be a dependency seeing as you reference it here?